src/Proxy.h

```diff
@@ -97,10 +97,20 @@ namespace sdbus::internal {
             using SignalName = std::string;
             struct SignalData
             {
+                SignalData(Proxy& proxy, signal_handler callback, SlotPtr slot)
+                    : proxy_(proxy)
+                    , callback_(std::move(callback))
+                    , slot_(std::move(slot))
+                {}
+                Proxy& proxy_;
```

Comment:
sdbus-c++ has been based on a convention without underscore in simple POD structs, and with underscore for members of classes. If we'd like to stick to that, I'd suggest that we remove underscore in member names here.